.github/workflows/swift-analyzer.py

```
@@ -0,0 +1,215 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2822
```

Ruff (0.11.9)

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found '{'

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ',', found ';'

---

SyntaxError: Expected ',', found '{'

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found ';'

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found ';'

---

SyntaxError: Expected ',', found '{'

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected ',', found ';'

---

SyntaxError: Expected ',', found ';'

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected '}', found 'import'

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Unexpected indentation

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected an f-string element or the end of the f-string

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected an f-string element or the end of the f-string

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected an f-string element or the end of the f-string

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected an f-string element or the end of the f-string

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected an f-string element or the end of the f-string

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Expected a newline after line continuation character

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Simple statements must be separated by newlines or semicolons

---

SyntaxError: Compound statements are not allowed on the same line as simple statements

---

Multiple statements on one line (colon)

(E701)